sale_order_line_available_qty/models/sale_order_line.py

```diff
@@ -0,0 +1,40 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright (C) 2015 Savoir-faire Linux
+#    (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields
+import openerp.addons.decimal_precision as dp
+
+
+class SaleOrderLine(models.Model):
+    _inherit = 'sale.order.line'
+
+    standard_price = fields.Float(
+        related='product_id.standard_price',
+        string='Cost Price',
+        digits_compute=dp.get_precision('Product Price'),
+    )
+
+    qty_available = fields.Float(
+        related='product_id.qty_available',
+        string='Available Quantity',
+        digits_compute=dp.get_precision('Product UoS'),
```

Comment:
Is this field correct when the warehouse is changed on a sales order? (by correct I mean, it shows the quantity in the selected warehouse)